### PR TITLE
[plg_content_pagenavigation]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_content_pagenavigation
### Testing Instructions

code review

### CODE REVIEW

File: plugins/content/pagenavigation/pagenavigation.xml [hunk 1]
```diff
@@ -21,30 +21,53 @@
...
 						<option value="1">PLG_PAGENAVIGATION_FIELD_VALUE_ARTICLE</option>
 						<option value="0">PLG_PAGENAVIGATION_FIELD_VALUE_TEXT</option>
 				</field>
-				<field name="display" type="list"
-					   default="0"
-					   label="PLG_PAGENAVIGATION_FIELD_DISPLAY_LABEL"
-					   description="PLG_PAGENAVIGATION_FIELD_DISPLAY_DESC"
+
+				<field
+					name="display" type="list"
```
Comment (brianteeman) [inline plugins/content/pagenavigation/pagenavigation.xml]: shouldnt this be on two lines?

File: plugins/content/pagenavigation/pagenavigation.xml [hunk 2]
```diff
@@ -21,30 +21,53 @@
...
+					name="position"
+					type="list"
 					label="PLG_PAGENAVIGATION_FIELD_POSITION_LABEL"
+					description="PLG_PAGENAVIGATION_FIELD_POSITION_DESC"
+					default="1"
 				>
-					<option value="1">PLG_PAGENAVIGATION_FIELD_VALUE_BELOW</option>
-					<option value="0">PLG_PAGENAVIGATION_FIELD_VALUE_ABOVE</option>
+					<option
+						value="1"
```
Comment (andrepereiradasilva) [inline plugins/content/pagenavigation/pagenavigation.xml]: the changes in the options are not needed. Is not a very big line and makes it harder to read IMO.

File: plugins/content/pagenavigation/pagenavigation.xml [hunk 3]
```diff
@@ -21,30 +21,53 @@
...
+				<field
+					name="display" type="list"
+					default="0"
+					label="PLG_PAGENAVIGATION_FIELD_DISPLAY_LABEL"
+					description="PLG_PAGENAVIGATION_FIELD_DISPLAY_DESC"
 				>
-					<option value="0">PLG_PAGENAVIGATION_FIELD_VALUE_NEXTPREV</option>
-					<option value="1">PLG_PAGENAVIGATION_FIELD_VALUE_TITLE</option>
+					<option
+						value="0"
```
Comment (andrepereiradasilva) [inline plugins/content/pagenavigation/pagenavigation.xml]: the changes in the options are not needed. Is not a very big line and makes it harder to read IMO.

Comment (matrikular) [inline plugins/content/pagenavigation/pagenavigation.xml]: I aggree with @andrepereiradasilva - though the changes logging OK on first sight, pretty please don't do this with the option fields. Options fields aren't usually that long and, especially in this case, it becomes harder to write and / or format. Apart from that, please keep the indentation to one tab.

File: plugins/content/pagenavigation/pagenavigation.xml [hunk 4]
```diff
@@ -21,30 +21,39 @@
...
-				<field name="position" type="list"
-					default="1"
-					description="PLG_PAGENAVIGATION_FIELD_POSITION_DESC"
+				<field
+					name="position"
+					type="list"
 					label="PLG_PAGENAVIGATION_FIELD_POSITION_LABEL"
+					description="PLG_PAGENAVIGATION_FIELD_POSITION_DESC"
+					default="1"
 				>
```
Comment (JoomliC) [inline plugins/content/pagenavigation/pagenavigation.xml]: Missing one tab indent

File: plugins/content/pagenavigation/pagenavigation.xml [hunk 5]
```diff
@@ -21,30 +21,39 @@
...
-						description="PLG_PAGENAVIGATION_FIELD_RELATIVE_DESC"
-						label="PLG_PAGENAVIGATION_FIELD_RELATIVE_LABEL"
+
+				<field
+					name="relative"
+					type="list"
+					label="PLG_PAGENAVIGATION_FIELD_RELATIVE_LABEL"
+					description="PLG_PAGENAVIGATION_FIELD_RELATIVE_DESC"
+					default="1"
 				>
```
Comment (JoomliC) [inline plugins/content/pagenavigation/pagenavigation.xml]: Missing one tab indent

File: plugins/content/pagenavigation/pagenavigation.xml [hunk 6]
```diff
@@ -21,30 +21,39 @@
...
-					   label="PLG_PAGENAVIGATION_FIELD_DISPLAY_LABEL"
-					   description="PLG_PAGENAVIGATION_FIELD_DISPLAY_DESC"
+
+				<field
+					name="display"
+					type="list"
+					label="PLG_PAGENAVIGATION_FIELD_DISPLAY_LABEL"
+					description="PLG_PAGENAVIGATION_FIELD_DISPLAY_DESC"
+					default="0"
 				>
```
Comment (JoomliC) [inline plugins/content/pagenavigation/pagenavigation.xml]: Missing one tab indent

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 8b51554c58164275843af1a9652ae5a1b9e6010d<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10474">issues.joomla.org/joomla-cms/10474</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 8b51554c58164275843af1a9652ae5a1b9e6010d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10474">issues.joomla.org/joomla-cms/10474</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10474">issues.joomla.org/joomla-cms/10474</a>.</sub>

**Pull request merged**